### ISSUE TITLE
Update feedstock once an official release is ready

### ISSUE THREAD

**Issue by emiliom** [conda-forge/cgsn_parsers-feedstock#1]
I've asked the `cgsn_parsers` package owner(s) to create an official release on their bitbucket repo,  https://bitbucket.org/ooicgsn/cgsn-parsers

Once that's ready, please update this feedstock to generate an updated conda package.

Also, as mentioned over email, please remove bad, older package duplicate named `cgsn-parsers` (dash instead of underscore).

cc @ocefpaf @rsignell-usgs

**Comment by ocefpaf** [comment]
> Also, as mentioned over email, please remove bad, older package duplicate named cgsn-parsers (dash instead of underscore).

Already done!

> Once that's ready, please update this feedstock to generate an updated conda package.

Please let us know when that happens. I have bots to follow PyPI and GitHub, but not bitbucket.

**Comment by emiliom** [comment]
Thanks. I'll follow up here once I hear about progress towards a release.

**Comment by emiliom** [comment]
Restarting this issue, based on second-hand word from @cwingard that the Bitbucket repo is, um, "ready to go" for conda packaging. Chris can chime in directly to explain what that means.

My role here is just as cheerleader and instigator. The conversation will be mainly between @ocefpaf (Filipe) and @cwingard -- and possibly @rsignell-usgs ?

Thanks!

**Comment by cwingard** [comment]
As far as "ready-to-go" goes, I've setup versioning with tags for the 
master branch and an initial pipeline (equivalent to travisci) for 
automated build-testing. If there is more I need to do to get this 
package ready for conda-forge, let me know.

Also, I removed the github version of this repo. The current version of 
the repo is at 0.4.1.

Cheers,
Chris

**Comment by emiliom** [comment]
Thanks, Chris!

@ocefpaf if you have any questions ... ask @cwingard :smile_cat: Thanks.

**Comment by ocefpaf** [comment]
I am getting this when trying to access the latest tarball

![screenshot from 2018-01-09 14-01-21](https://user-images.githubusercontent.com/950575/34731208-6df221b4-f540-11e7-9ff9-a38446132cfe.png)

I'll try to submit a PR anyway, maybe only the bitbucket South America mirrors are down.

**Comment by ocefpaf** [comment]
Nope. It did not work,

![screenshot from 2018-01-09 14-33-36](https://user-images.githubusercontent.com/950575/34731723-f10ed190-f541-11e7-867d-d2815eed9d1f.png)


We'll have to wait for bitbucket to stabilize its services first.

**Comment by emiliom** [comment]
Woo-hoo! Thank you @ocefpaf and @cwingard !!

@lsetiawan give it a try on our NANOOS environment later this week, say, Friday.
